.builders/images/linux-x64_86/Dockerfile

```
@@ -3,6 +3,15 @@ FROM ${BASE_IMAGE}
 
 ARG PYTHON_MAJOR
 
+ENV IBM_MQ_VERSION 9.2.4.0
+ENV IBM_MQ_SHA256 "d0d583eba72daf20b3762976f8831c2e23150ace90509520e12f8cda5b5bdb49"
```

Omitting the equals sign may be dropped in future https://docs.docker.com/engine/reference/builder/#env

```suggestion
ENV IBM_MQ_VERSION="9.2.4.0"
ENV IBM_MQ_SHA256="d0d583eba72daf20b3762976f8831c2e23150ace90509520e12f8cda5b5bdb49"
```